<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dlg_Login</class>
 <widget class="QDialog" name="Dlg_Login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>335</width>
    <height>270</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>335</width>
    <height>270</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>335</width>
    <height>270</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>登录</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/*QPushButton[name=&quot;btn&quot;]{border-radius:4px;background:#409eff;color:white;font-size:14px;min-height:24px;min-width:85px;padding:4px;}
QPushButton[name=&quot;btn&quot;]:pressed{background:#3a8eed}

QLineEdit[name=&quot;le&quot;]{border-radius:4px;min-height:25px;border:1px solid grey;max-width:280px;min-width:280px;}

QLabel[name=&quot;title&quot;]{font:18px '黑体';color:blue;}

QWidget[name=&quot;bg&quot;]{background-color:white;}*/
QLabel[name=&quot;lb&quot;]{color:#2D77E5;}
</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="2" column="0">
    <widget class="QWidget" name="widget" native="true">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>50</height>
      </size>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="verticalSpacing">
       <number>0</number>
      </property>
      <item row="0" column="1">
       <widget class="QPushButton" name="btn_exit">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>退出</string>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
        <property name="name" stdset="0">
         <string>btn</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup</string>
        </attribute>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QPushButton" name="btn_login">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>登录</string>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
        <property name="name" stdset="0">
         <string>btn</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup</string>
        </attribute>
       </widget>
      </item>
     </layout>
     <zorder>btn_login</zorder>
     <zorder>btn_exit</zorder>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QWidget" name="widget_2" native="true">
     <layout class="QGridLayout" name="gridLayout_3">
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="lb_username">
        <property name="text">
         <string>用户名</string>
        </property>
        <property name="name" stdset="0">
         <string>smtitle</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="le_username">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="name" stdset="0">
         <string>le</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lb_userpass">
        <property name="text">
         <string>密码</string>
        </property>
        <property name="name" stdset="0">
         <string>smtitle</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="le_userpass">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="name" stdset="0">
         <string>le</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="lb_title">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>70</height>
      </size>
     </property>
     <property name="text">
      <string>个人通讯录</string>
     </property>
     <property name="name" stdset="0">
      <string>title</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
